client/src/client_proxy.rs

```diff
@@ -802,7 +802,7 @@ impl ClientProxy {
         Ok(())
     }
 
-    /// Recover wallet accounts from file and return vec<(account_address, index)>.
+    /// Recover wallet accounts from space_delim_strings and return vec<(account_address, index)>.
```

Comment:
space_delim_strings should be written out "space delimited strings" or even better yet giving an example of the formatting.